docs/conf/modules.conf.example

```diff
@@ -1591,6 +1591,8 @@
 # Layer via AUTHENTICATE. Note: You also need to have m_cap.so loaded
 # for SASL to work.
 #<module name="m_sasl.so">
+# Define the following to your Services server name to be more secure.
```

Comment:
Services should be lower case here as its a common noun not a proper noun.

Also, you should say /why/ using this setting is more secure rather than just saying that it is more secure.

Reply:
Fixed/updated